lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -146,6 +146,144 @@ def formatted_message_and_backtrace(colorizer, indentation)
+          end
+
+          def multiple_failure_sumarizer(exception, prior_detail_formatter, color)
+            lambda do |example, colorizer, indentation|
+              summary = if exception.aggregation_metadata[:from_around_hook]
+                          "Got #{exception.exception_count_description}:"
+                        else
+                          "#{exception.summary}."
+                        end
+
+              summary = colorizer.wrap(summary, color || RSpec.configuration.failure_color)
+              return summary unless prior_detail_formatter
+              "#{prior_detail_formatter.call(example, colorizer, indentation)}\n#{indentation}#{summary}"
+            end
+          end
+
+          def sub_failure_list_formatter(exception, message_color)
+            common_backtrace_truncater = CommonBacktraceTruncater.new(exception)
+
+            lambda do |failure_number, colorizer, indentation|
+              exception.all_exceptions.each_with_index.map do |failure, index|
+                options = with_multiple_error_options_as_needed(
+                  failure,
+                  :description_formatter   => :failure_slash_error_line.to_proc,
+                  :indentation             => indentation.length,
+                  :message_color           => message_color || RSpec.configuration.failure_color,
+                  :skip_shared_group_trace => true
+                )
+
+                failure   = common_backtrace_truncater.with_truncated_backtrace(failure)
+                presenter = ExceptionPresenter.new(failure, @example, options)
+                presenter.fully_formatted("#{failure_number}.#{index + 1}", colorizer)
+              end.join
+            end
+          end
+
+          # @private
+          # Used to prevent a confusing backtrace from showing up from the `aggregate_failures`
+          # block declared for `:aggregate_failures` metadata.
+          module EmptyBacktraceFormatter
+            def self.format_backtrace(*)
+              []
+            end
+          end
+
+          # @private
+          class CommonBacktraceTruncater
+            def initialize(parent)
+              @parent = parent
+            end
+
+            def with_truncated_backtrace(child)
+              child_bt  = child.backtrace
+              parent_bt = @parent.backtrace
+              return child if child_bt.nil? || child_bt.empty? || parent_bt.nil?
+
+              index_before_first_common_frame = -1.downto(-child_bt.size).find do |index|
+                parent_bt[index] != child_bt[index]
+              end
+
+              return child if index_before_first_common_frame == -1
+
+              child = child.dup
+              child.set_backtrace(child_bt[0..index_before_first_common_frame])
+              child
+            end
+          end
+        end
+
+        # @private
+        PENDING_DETAIL_FORMATTER = Proc.new do |example, colorizer|
```

Comment:
It's meant to be the value used for the `detail_formatter` option (defined [here](https://github.com/rspec/rspec-core/blob/5375dcdfdb5460a51d5003de76eb124997fa26ea/lib/rspec/core/formatters/exception_presenter.rb#L15)) for pending specs.  In fact, the exception presenter has multiple options with the `_formatter` suffix.

If you want to come up with better names for all of them, that would be fine, but it didn't particularly bother me since they aren't classes (and thus don't have the implication that a `XYZFormatter` class does).  I'd prefer that we either change all of them to not have the `_formatter` or leave them all as is.